program/convert-ssd-to-i8/README.md

```diff
@@ -0,0 +1,52 @@
+# convert-ssd-to-i8
+
+Program for conversion of SSD models into Intel Caffe quantized format.
+
+## Requirements
+
+Intel Caffe library:
+```
+ck install package --tags=lib,caffe,intel
+```
+
+Caffe model to be converted into quantized format:
+```
+ck install package --tags=caffemodel,ssd
+```
+
+One of datasets COCO-2014 or VOC-2207 depending on wich model you want convert:
```

Comment:
depending on which model you wish to convert:

Reply:
Changed.